onnxruntime/python/tools/transformers/huggingface_models.py

```diff
@@ -0,0 +1,111 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.  See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+# List of pretrained models: https://huggingface.co/transformers/pretrained_models.html
+# Pretrained model name to a tuple of input names, opset_version, use_external_data_format and optimization model type
+MODELS = {
+# BERT
+    "bert-base-uncased": (["input_ids", "attention_mask", "token_type_ids"], 11, False, "bert"),
```

Comment:
nit: no much value to add so many bert models that quite similar. 

I think the following are typical:
bert-base-cased, bert-large-cased, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc

Reply:
Verified that exported squad model has two outputs